src/main/java/org/kiwiproject/migrations/AbstractMongockCommand.java

```diff
@@ -0,0 +1,43 @@
+package org.kiwiproject.migrations;
```

Comment:
Would it make sense to put all this under one more `'mongo` subpackage? i.e. `org.kiwiproject.migrations.mongo`?

@terezivy ?